src/main.cpp

```diff
@@ -63,8 +64,15 @@ int main() {
 
     // Main game loop
     Vector3 pos;
+    raylib::AudioDevice audio;
+    raylib::Music music(MUSIC_CONFIG_PATH + "Dubioza kolektiv - Free.mp3 (The Pirate Bay Song).wav");
```

Comment:
So, I like how you started on a config file for the music stuff. I think a nice API for the music is if we could do something like ASound::song("menu") and it would switch to the appropriate song specified in the config file, or ASound::play("jump") and it would play the sound effect

Reply:
so, that's possible for sound effects, but for background music, we need to update the stream in the game loop

see https://github.com/RobLoach/raylib-cpp/blob/master/examples/audio/audio_music_stream.cpp#L40

Reply:
I did it!